README.md

```diff
@@ -348,15 +382,27 @@ If any of the following criteria are not met, the UCAN MUST be considered invali
 
 A UCAN's time bounds MUST NOT be considered valid if the current system time is prior to the `nbf` field, or after the `exp` field. This is called "ambient time validity".
 
-All witnesses MUST contain time bounds equal to or wider than the UCAN being delegated to. If the witness expires before the outer UCAN — or starts after it — the reader MUST treat the UCAN as invalid. This is called "timely delegation".
+All witnesses MUST contain time bounds equal to or wider than the UCAN being delegated to. If the witness expires before the outer UCAN — or starts after it — the reader MUST treat the UCAN as invalid. This is called "timely delegation". This conditions MUST hold even if the current wall clock time is inside the misdelegated bounds. 
```

Comment:
Typo: "This conditions" -> "These conditions"

Reply:
Thanks!

)I'm going to run this through a spell checker before flipping it out of Draft)

Reply:
Whoops, sounds good! I was taking a break from reading and saw the notifications came in so I took a quick look :)

Reply:
No problem — seriously thank you! Mainly there's going to be a bunch of these, and we can let a computer fix most of my late night typos 😛 I would love a proof read when it's ready, though!

Reply:
Sounds good! Mostly I'm trying to stay on top of the changes because each iteration gives me more context about the spec, and forms a clearer picture of it in my head